openff/toolkit/topology/molecule.py

```diff
@@ -6125,7 +6128,7 @@ def has_unique_atom_names(self) -> bool:
         """``True`` if the element has unique atom names, ``False`` otherwise."""
         return _has_unique_atom_names(self)
 
-    def generate_unique_atom_names(self):
+    def generate_unique_atom_names(self, suffix: str = None):
```

Comment:
```suggestion
    def generate_unique_atom_names(self, suffix: str = "x"):
```
Again, same as above